### ISSUE TITLE
Inspector Docs page in dark mode has very low contrast ratio -- hard to see commands

### ISSUE THREAD

**Issue by thawsitt** [kamilkisiela/graphql-inspector#1937]
Summary: It is hard to see code commands in [docs page](https://graphql-inspector.com/docs/essentials/validate) in dark mode. No way to switch back to light mode.

**To Reproduce**
- Go to https://graphql-inspector.com/docs/essentials/validate
- Check "Documents" and "Schema" code blocks. It is very hard to see blue text against grey background. Similar issue can be seen in other pages. See screenshot below:

![Low contrast ratio](https://user-images.githubusercontent.com/13971250/116735384-43f7aa80-a9a3-11eb-8551-788fefb08dce.png)

**Expected behavior**

The text should be legible and has enough contrast ratio.

**Additional context**

Is there a way to switch between light and dark mode? I also tried Incognito mode: it shows light mode first but automatically switches to dark mode when I refresh the page.

Browsers: Chrome v90, Firefox v88

**Comment by thawsitt** [comment]
The issue seems to be fixed now by these commits:

- c44309f46809918f8a64b8b056792fd282d746dd
- c44309f46809918f8a64b8b056792fd282d746dd

Thanks @kamilkisiela! ❤️

**Comment by kamilkisiela** [comment]
:)
